src/themes/slider/theme.c

```diff
@@ -383,31 +382,30 @@ GtkWindow* create_notification(UrlClickedCb url_clicked)
 	gtk_widget_show(windata->main_hbox);
 	gtk_box_pack_start(GTK_BOX(main_vbox), windata->main_hbox, FALSE, FALSE, 0);
 
-	/* First row (icon, vbox, close) */
-	windata->iconbox = gtk_alignment_new(0.5, 0, 0, 0);
```

Comment:
Looks like iconbox variable isn't used anymore after this commit, so it can be removed.